tutorials/katacoda/thanos/2-lts/intro.md

```diff
@@ -1,7 +1,14 @@
-# In progress
+[Thanos](thanos.io) is a set of components that can be composed into a highly available metric system with unlimited storage capacity. It can be added seamlessly on top of existing Prometheus deployments.
 
-🚧 This tutorial is in progress. 🚧
+Thanos is known as long-term storage for Prometheus.
 
-Do you want to see this soon or you want to help us?
+Thanos works in cloud native environments like Kubernetes as well as traditional ones. This course uses docker containers with pre-built docker images.
 
-Let us know on https://github.com/thanos-io/thanos or #thanos slack channel linked on https://thanos.io
\ No newline at end of file
+In this tutorial, you will learn about :
```

Comment:
Worth to mention something like "If you did not see previous tutorials, please go through them now to understand this part better!"